README.md

```diff
@@ -55,7 +55,7 @@ jobs:
     runs-on: ubuntu-22.04
     steps:
       - name: Build and push to ECR
-        uses: citizensadvice/build-and-push-action@v1
+        uses: citizensadvice/build-and-private-ecr-push-action@v1
```

Comment:
@Shaqil1989 this shouldn't change. It is an example of how to to use the action in this repo, not the one you renamed

Reply:
This was the wrong repository, i have changed this back.